src/main/realtime.ts

```diff
@@ -12,16 +15,22 @@ function cleanAndStartRemoteNotifications() {
   stopRemoteNotifications();
 
   socket = io(process.env.NOTIFICATIONS_URL, {
+    transports: ['websocket', 'polling'],
     auth: {
       token: obtainToken('bearerToken'),
     },
     withCredentials: true,
   });
 
-  socket.io.on('open', () => {
+  socket.on('open', () => {
```

Comment:
Why is now directly listening from the socket and not from io?

Reply:
I believe this was incorrect before, socket.io is a readonly property with underlying logic (they call it a Manager, it holds all the connection logic), socket is a class that extends from EventEmitter, idk if they are the same, but since the rest of the methods uses `socket.on` instead of `socket.io.on` I decided to move it to `socket.on`